src/components/ContentSidebar/ActivityFeed/approval-comment-form/ApprovalCommentForm.scss

```diff
@@ -1,6 +1,8 @@
 @import '../../../variables';
 
 .be .bcs-comment-input-container {
+    @import '~box-react-ui/lib/styles/common/overlay';
```

Comment:
Do all overlay styles needed?

Reply:
We need all except one small toggle one at the bottom so it made more sense to just import the whole thing